.github/workflows/e2e-chaos.yaml

```diff
@@ -96,6 +96,8 @@ jobs:
           make hack/benchmark/assets/dataset/${DATASET}
           go version
           go test \
+            -shuffle=on \
```

Comment:
I think the e2e tests should not be shuffled.

Reply:
can you tell the the reason why do you think that?

Reply:
The e2e tests depends on the state of the external objects (= Vald Agent/Cluster), not in the test codes. Their states depends on the order of the operations.